src/fear_and_greed/cnn.py

```diff
@@ -62,3 +53,24 @@ def get(fetcher: Fetcher = None) -> FearGreedIndex:
         description=response["rating"],
         last_update=datetime.datetime.fromisoformat(response["timestamp"]),
     )
+
+def historical(fetcher: Fetcher = None) -> dict:
+    """Returns CNN's Fear & Greed Index historical data."""
+
+    if fetcher is None:
+        fetcher = Fetcher()
+
+    response = fetcher()["fear_and_greed_historical"]
+    fear_greed_historical = []
+    
+    if "data" in response:
```

Comment:
Should we raise an exception if `data` isn't found?

Reply:
I agree, see new commits